charts/llm-d-modelservice/values.yaml

```diff
@@ -7,6 +7,9 @@
 # -- Common configuration values
 common: {}
 
+# -- Usually used when using llm-d-modelservice as a subchart
+enabled: true
```

Comment:
Can we remove this? This field is mainly used as a `condition` in `chart.yaml` whcn as subchart, and by default, this field is set to `true` in the schema.json